features/support/fs.rs

```diff
@@ -97,6 +101,10 @@ pub fn create_project() -> io::Result<Project> {
 }
 
 fn create_cargo_toml(dir: TempDir) -> io::Result<TempDir> {
+  fn double_backslash<T: Display>(path: T) -> String {
```

Comment:
1. Currently, the method is pretty specific (with it using the Display trait and naively doubling \ ), so I thought it might be reasonable to hide it from general access - although I don't know when it's idiomatic in Rust to use inner methods.
2. I didn't get the code to compile otherwise - is there a way to coerce it from Display -> String automatically?